src/App.tsx

```diff
@@ -14,6 +36,132 @@ export const App: React.FC = () => {
     }
   }, []);
 
+  const getTodosFromApi = useCallback(async (userId: number) => {
+    try {
+      const response = await getTodos(userId);
+
+      setTodos(response);
+      setTempTodo(null);
+      setIsEditing(null);
+      setIsAllSelected(response.every(todo => !todo.completed));
+    } catch (error) {
+      setErrorMessage('Unable to load todos');
+      setIsHidden(false);
+    }
+  }, []);
+
+  const createTodo = async (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (!inputTitle) {
+      setErrorMessage('Title can\'t be empty');
+      setIsHidden(false);
+
+      return;
+    }
+
+    if (user) {
```

Comment:
Start your function with `if (!user) return` - it's better way to avoid unnecessary execution. Also no need to wrap a lot of code in block.